calico-cloud_versioned_docs/version-19-1/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -53,7 +53,7 @@ To deploy the registry scanner as a pod in Kubernetes cluster, we recommend that
 
 ### Download the registry scanner
 
-The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:{{imageassuranceversion}}`.
+The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.15.3`.
```

Comment:
I think it's worth adding a ToDo comment just to mark down that we're explicitly coming back here to switch it to the `imageassuranceversion` parameter on the next release.

Reply:
No need. Next release will be cut from ./calico-cloud/, where we are not making the change.